### ISSUE TITLE
Add "Open in Google Maps" functionality

### ISSUE THREAD

**Issue by troxler** [mapsme/omim#4464]
Sometimes maps.me does not have all the information you need about certain area or POI (e.g. satellite view, POI ratings, images, etc.). Please add an "Open in Google Maps" button that will open either the area or (if selected) the POI in Google Maps.

**Comment by filutkie** [comment]
Yes, the Maps.me doesn't provide any satellite views, reviews and photos of places but I think it is kind of irrelevant for the project based on the OpenStreetMap data. Why use Google Maps as default? Why not Here or Yandex? And Apple users might not have Gmaps installed at all. 

Maps.me does it work as it should for the best offline maps app. **Google Maps has its own offline mode**, where you can browse a POI rating, cached tiles etc.

Moreover Maps.me is supposed to be used offline without internet connection mostly and there is no reason to open another internet-dependent maps application.

**Comment by rugk** [comment]
I agree with @filutkie, there is no need to add a button for opening in GMaps.
Maybe a button "Open in another app" or "Share to another app", which just passes the `geo:` URI (or how it is handeled) to Android. In this case Android most likely offers a list of apps you can use to open the URI. This would be useful IMHO. (And it would fulfil @troxler usecase.)

**Comment by troxler** [comment]
@rugk You are right, this should not just work for Google Maps but for any map application that can handle such URIs.

**Comment by burivuh** [comment]
It is not a very good business case for us to open another maps application. But we have a plan of integrating "additional value" services like Foursquare in 2017.

**Comment by rugk** [comment]
That's okay, if you can always disable it.

**Comment by thany** [comment]
@burivuh
Users couldn't care less about business plans. They just want the best experience (and so should you).

**Comment by rpodgorny** [comment]
any news on this one?

**Comment by thany** [comment]
Thank goodness that "additional value" crap didn't make it.

As for the original issue, I would still like this as well. It just makes sense. Why would you take your data from OSM, and then complain about business value? The data is open, the software is open-source, and so if folks want a certain feature, brushing it off with "business value" blahblah is counterproductive.

However, knowing the incredible stubbornness of maps.me developers, this feature will never make it in. Just like so many other things that take less time to build/fix than this one. Thumbs-down if you must, but it has been true for years, and I see no sign of change.
